docs/quickstart-docker.md

```diff
@@ -5,93 +5,99 @@ Percona XtraBackup works in combination with a MySQL-compatible database, Percon
+| `backupvol` | Indicates the persistent storage for the database. |
+| `psmysql`    | Indicates the Percona Server for MySQL container.  |
+|`--password`   | Prompts the user to enter the password for the root user. For convenience you can add the password for the root user to this option, for example, `--password=secret`. Then the password will be passed automaticaly when running the command. Note that if you specify the password with `--password=secret`, the password is visible in `docker ps`, `docker ps -a` (docker history) and regular ps command.          |
+| `8.0.34` | Use this tag to specify a specific version. Avoid using the `latest` tag. <br> In our example, we use the `8.0.34` tag. In Docker, a tag is a label assigned to an image and is used to maintain different versions of an image.|
 
-    | Option | Description |
-    |---|---|
-    | `--name`  |  Provides a name to the container. If you do not use this option, Docker adds a random name.  |
-    | `--volumes-from` | Refers to Percona Server for MySQL and indicates that you intend to use the same data as the `psmysql` container.|
-    | `it` | Interacts with the container and be a pseudo-terminal. |
-    | `--user root` | Sets the user to root inside the Percona XtraBackup container. This option is required to access the MySQL data directory and run the xtrabackup command. |
-    |`backup_8034` |Indicates the directory inside the container where the backup files are stored. |
-    | `-v` | Mounts a volume from the host machine to volumes in another container that is being run. In our example, the `-v` option mounts a volume from the `psmysql` container to the `backupvol` volume on the host machine.|
-    | `backupvol` | Indicates the persistent storage for the database. |
-    | `psmysql`    | Indicates the Percona Server for MySQL container.  |
-    |`--password`   | Prompts the user to enter the password for the root user. For convenience you can add the password for the root user to this option, for example, `--password=secret`. Then the password will be passed automaticaly when running the command. Note that if you specify the password with `--password=secret`, the password is visible in `docker ps`, `docker ps -a` (docker history) and regular ps command.          |
-    | `8.0.34` | Use this tag to specify a specific version. Avoid using the `latest` tag. <br> In our example, we use the `8.0.34` tag. In Docker, a tag is a label assigned to an image and is used to maintain different versions of an image.|
+If you do not add a tag, Docker uses `latest` as the default tag and downloads the newest image from [percona/percona-xtrabackup on the Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup). This image can be in a different series or version from what you expect since the latest image changes over time. The usage of tags ensures that you use [compatible versions](quickstart-overview.md#limitations) of Percona Server for MySQL and Percona XtraBackup.
```

Comment:
But if you are using 8.0.34 or later you should be able to make a backup. 
See supported versions.

Reply:
Updated